src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -22,16 +20,15 @@
     <PublishSingleFile Condition="'$(NativeAotSupported)' != 'true'">true</PublishSingleFile>
     <PublishTrimmed Condition="'$(NativeAotSupported)' != 'true'">true</PublishTrimmed>
     <SuppressGenerateILCompilerExplicitPackageReferenceWarning>true</SuppressGenerateILCompilerExplicitPackageReferenceWarning>
-    <!-- Compute host package name (taken from Microsoft.DotNet.ILCompiler.SingleEntry.targets) -->
+    <!-- Compute host OS and architecture (taken from Microsoft.DotNet.ILCompiler.SingleEntry.targets) -->
     <_hostOS>$(NETCoreSdkPortableRuntimeIdentifier.SubString(0, $(NETCoreSdkPortableRuntimeIdentifier.LastIndexOf('-'))))</_hostOS>
```

Comment:
`_hostOS` is globally defined. https://github.com/dotnet/runtime/blob/main/Directory.Build.props#L17-L24. Can this be deleted?